tinyagentos/routes/skill_exec.py

```
@@ -225,6 +225,16 @@ async def _skill_notify_user(args: dict, request: Request) -> dict:
         return {"error": str(exc)}
 
 
+async def _skill_list_frameworks(args: dict, request: Request) -> dict:
+    """List the agent frameworks taOS can deploy (so the agent can offer one)."""
+    try:
+        from tinyagentos.tools.framework_tools import execute_list_frameworks
+
+        return await execute_list_frameworks(args, request)
+    except Exception as exc:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)